config/locales/en/providers.yml

```diff
@@ -62,16 +62,17 @@ en:
             the prospects of success
     check_benefits:
       index:
-        heading_1: Your client's tax and benefits status
-        heading_2: What happens next
+        next_heading: What happens next
         negative_result:
-          body: Your client does not receive benefits that qualify for legal aid.
-          next: Your client will have to complete a financial means assessment. They can complete the financial assessment online from home or another location.
           title: must complete a financial assessment
+          paragraph: 'This is because they do not receive benefits that qualify for legal aid.'
+          next_1: Your client will need to answer questions about their financial situation.
+          next_2: 'We''ll ask you to give them access to the online assessment which they can complete at home or at another location.'
         positive_result:
-          body: Your client receives benefits that qualify for legal aid.
-          next: Your client will have to declare whether they have any savings, own a home or anything else that can pay for legal aid.
-          title: receives a qualifying benefit
+          title: receives benefits that qualify for legal aid
+          paragraph: 'You''ll need to:'
```

Comment:
Would this be better as ```You will need to``` instead of ```You'll```?  GDS suggests avoiding contractions

Reply:
This is one for Jim.  He, Justin, Vero etc are all looking at this ticket again.  So there might be some changes like that coming...

Reply:
FYI, if you use double quotes you won't have to escape the apostrophe.
`paragraph: "You'll need to:"`

Reply:
They have re-looked at this and no changes.  

You can also see Jim is happy to go ahead with the current wording. So I guess this contraction is okay.

Reply:
I was just copying previous notation of single quotes and escaping that is used elsewhere.

Reply:
YAML automatically converts the value to a string so you don't need put the text into quotes unless there is a special character in the string (for example `:`) which YAML has problems parsing. So:
```yaml
next_2: We'll ask you to give them access to the online assessment which they can complete at home or at another location.
paragraph: "You'll need to:"
```